examples/user_guide/08-Gridded_Datasets.ipynb

```diff
@@ -30,6 +31,17 @@
     "* they have coordinate or key dimensions, which describe the sampling of each dimension in the value arrays\n",
     "* they have value dimensions which describe the quantity of the multi-dimensional value arrays\n",
     "\n",
+    "There are many different types of gridded datasets defined by their coordinates, however in holoviews gridded datasets mostly refers to two types of grids usually known as regular and irregular coordinates or rectilinear or curvilinear. Rectilinear grids are defined by 1D coordinate arrays while rectilinear grid coordinates have the same dimensionality as the underlying value arrays. Among rectilinear grids there are two different subtypes, uniform and irregularly spaced grid. Only some types of elements support irregularly spaced rectilinear and curvilinear grids, specifically a QuadMesh is usually used to represent these.\n",
```

Comment:
```suggestion
    "There are many different types of gridded datasets, which each approximate or measure a surface or space at discretely specified coordinates.  In HoloViews, gridded datasets are typically one of three possible types: Regular rectilinear grids, irregular rectilinear grids, and curvilinear grids.  Regular rectilinear grids can be defined by 1D coordinate arrays specifying the spacing along each dimension, while the other types require grid coordinates with the same dimensionality as the underlying value arrays, specifying the full n-dimensional coordinates of the corresponding array value. HoloViews provides many different elements supporting regularly spaced rectilinear grids, but currently only QuadMesh supports irregularly spaced rectilinear and curvilinear grids.\n",
```

(Please edit as needed; I'm not certain this captures the actual behavior, but hopefully you get the idea. In the original I was getting very confused about whether there were two types of grids each with two names, four types, three types, etc. Here I'm assuming there are three types, but if irregular curvilinear is also an option distinct from just curvilinear, then please add that as a fourth type.  )

Reply:
No you got it right. Initially I'd written it as two types, where uniform rectilinear are simply a subset of rectilinear grids in general.